clients/privacy-center/cypress/e2e/consent-banner-tcf.cy.ts

```diff
@@ -258,13 +275,34 @@ describe("Fides-js TCF", () => {
           .within(() => {
             cy.get("span").contains("IAB TCF");
           });
+        cy.get("span")
+          .contains(newVendor.name)
+          .within(() => {
+            cy.get("span").contains("IAB TCF");
+          });
 
         // Filter to just GVL
         cy.get(".fides-filter-button-group").within(() => {
           cy.get("button").contains("IAB TCF vendors").click();
         });
         cy.get("span").contains(SYSTEM_1.name).should("not.exist");
         cy.get("span").contains(VENDOR_1.name);
+        cy.get("span").contains(newVendor.name);
+
+        // Check that the vendor ids persisted to the TC string
+        cy.getByTestId("consent-modal").within(() => {
+          cy.get("button").contains("Opt in to all").click();
+        });
+        cy.window().then((win) => {
+          win.__tcfapi("getTCData", 2, cy.stub().as("getTCData"));
+        });
+        cy.get("@getTCData")
+          .should("have.been.calledOnce")
+          .its("lastCall.args")
+          .then(([tcData, success]) => {
+            expect(success).to.eql(true);
+            expect(tcData.vendor.consents).to.eql({ 1: true, 2: true });
```

Comment:
What does the `{1: true, 2: true}` correspond to?

Reply:
ah—those are vendor IDs, saying that we opted in to vendors 1 and 2 (the new gvl vendor I added was `gvl.1`, and the original vendor in the [fixture was `2`](https://github.com/ethyca/fides/blob/08aca0f5a2420f8cc83826997eb092d9c08a3398/clients/privacy-center/cypress/fixtures/consent/experience_tcf.json#L304)